libpod/oci_linux.go

```diff
@@ -418,7 +418,7 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string, res
 		logrus.Debugf("Received container pid: %d", ss.si.Pid)
 		if ss.si.Pid == -1 {
 			if ss.si.Message != "" {
-				return errors.Wrapf(ErrInternal, "container create failed: %s", ss.si.Message)
+				return errors.Wrapf(ErrInternal, ss.si.Message)
```

Comment:
Personally I wouldn't have made this change.

Reply:
This is in hold right now, while waiting for @giuseppe Patch to get merged, I actually want this to go further. 
This is not an Internal Err, and it is not failing to create the container.  The container was created successfully and the command failed, within the container.  So we are reporting two incorrect messages up the stack.